src/config-inferrer.ts

```diff
@@ -82,11 +90,16 @@ export class ConfigInferrer {
         return await ctx.exists('Makefile') || await ctx.exists('makefile');
     }
 
-    protected async checkMake(ctx: Context) {
-        if (await ctx.exists('CMakeLists.txt')) {
-            this.addCommand(ctx.config, 'cmake .', 'init');
-        } else if (await this.isMake(ctx)) {
-            this.addCommand(ctx.config, 'make', 'init');
+    protected async checkStack(ctx: Context) {
+        if (await ctx.exists("stack.yml") || await ctx.exists("stack.yaml")) {
+            this.addCommand(ctx.config, "stack install", "init");
```

Comment:
I think you mean `stack.yaml`, not `stack.yml`, right?

https://docs.haskellstack.org/en/stable/stack_yaml_vs_cabal_package_file/

Reply:
Also, why do you prefer it over `package.yaml` or `*.cabal`?

Reply:
Also please add a test for this in the spec.

Reply:
Wait `ctx.exists` supports globbing